Grbl_Esp32/src/Config.h

```diff
@@ -96,7 +96,7 @@ const int MAX_N_AXIS = 6;
 //#define CONNECT_TO_SSID  "your SSID"
 //#define SSID_PASSWORD  "your SSID password"
 //CONFIGURE_EYECATCH_BEGIN (DO NOT MODIFY THIS LINE)
-#define ENABLE_BLUETOOTH  // enable bluetooth
+//#define ENABLE_BLUETOOTH  // enable bluetooth
```

Comment:
Do we want to change this default as part of an unrelated patch?

Reply:
I don't know. It was just to speed up compiling during testing. I should have reverted it.